test/e2e/parallel/space_test.go

```diff
@@ -222,11 +222,11 @@ func TestSpaceRoles(t *testing.T) {
 	})
 
 	t.Run("set owner user as contributor instead", func(t *testing.T) {
-		// given an appstudio space with `owner` user as an admin of it
-		ownerBinding.Spec.SpaceRole = "contributor"
-
 		// when
-		err := hostAwait.Client.Update(context.TODO(), ownerBinding)
+		ownerBinding, err = hostAwait.UpdateSpaceBinding(t, ownerBinding.Name, func(sb *toolchainv1alpha1.SpaceBinding) {
```

Comment:
I was wondering if there something special with `UpdateSpaceBinding` and `UpdateSpace`, or they could be replaced by the new generic update function as well ? 

Not something we need to address in this PR anyways.

Reply:
As mentioned in the other comment, they will be replaced in the following PR https://github.com/codeready-toolchain/toolchain-e2e/pull/1074#issuecomment-2511409880

Reply:
Got it , I've missed that ! Thanks 👍